src/conways-game-of-life/index.ts

```diff
@@ -0,0 +1,81 @@
+const updateCellStatus = (x: number, y: number, currentCellStatus: number, neighbourSum: number) => {
```

Comment:
`x` and `y` are not used on this method. Remove the unnecessary parameters.

Reply:
I would have used them in a previous version, but I changed my mind and forgot to remove them. Thanks for pointing it out.